docs/sources/tempo/configuration/_index.md

```diff
@@ -532,7 +532,7 @@ metrics_generator:
     # This option only allows spans with end times that occur within the configured duration to be
     # considered in metrics generation.
     # This is to filter out spans that are outdated.
-    [metrics_ingestion_time_range_slack: <duration> | default = 30s]
+    [metrics_ingestion_time_range_slack: <duration> | default = 2m0s]
```

Comment:
let's leave this at 30s. i think a local Tempo OSS install would prefer 30s. the default of 2m for cloud is only due to the huge diversity of a clients

Reply:
reverted this change